python/tests/test_cli.py

```diff
@@ -263,29 +263,35 @@ def test_fasta_long_args(self):
         assert args.tree_sequence == tree_sequence
         assert args.wrap == 50
 
-    def test_vcf_default_values(self):
+    test_parameters_vcf_ploidy = (
```

Comment:
What's the benefit of a variable here? Might as well just put it in the call to parametrize.